plugin/src/main/java/net/dzikoysk/funnyguilds/guild/top/GuildTop.java

```diff
@@ -0,0 +1,22 @@
+package net.dzikoysk.funnyguilds.guild.top;
+
+import java.util.Comparator;
+import java.util.NavigableSet;
+import java.util.function.Function;
+import net.dzikoysk.funnyguilds.guild.Guild;
+import net.dzikoysk.funnyguilds.guild.GuildRank;
+import net.dzikoysk.funnyguilds.rank.Top;
+
+public class GuildTop extends Top<GuildRank> {
+
+    public GuildTop(Comparator<GuildRank> comparator, Function<Comparator<GuildRank>, NavigableSet<GuildRank>> recalculateFunction) {
+        super(comparator, recalculateFunction);
+    }
+
+    public Guild getGuild(int place) {
+        return this.get(place)
+                .map(GuildRank::getGuild)
+                .getOrNull();
+    }
```

Comment:
Not sure how we're using this function, but I think handling missing value could be more convenient for the caller with value wrapped into an Option here, as it could be used with `.map().orElse()`.